s3config.yaml.sample

```diff
@@ -70,6 +70,7 @@ S3_SERVER_CONFIG:
    S3_SERVER_MOTR_ETIMEDOUT_MAX_THRESHOLD: 100          # Number of ETIMEDOUT errors per monitoring window before s3server restart
    S3_SERVER_MOTR_ETIMEDOUT_WINDOW_SEC: 1               # Monitoring window for motr ETIMEDOUT errors in seconds
    S3_SERVER_ENABLE_ADDB_DUMP: true                     # If set to true, then addb dump will be collected
+   S3_MAX_EXTENDED_OBJECTS_IN_FAULT_MODE: 10            # Maximum number of extended objects created when S3 write IO fails
```

Comment:
Max number of allowed recovery by s3 when fault happens and motr error's out with specific error code

By default this will be 1
If its 0 then we dont do any recovery

Reply:
yes. If it is 0, the feature is disabled. This is how it is implemented. 
I'll add the text you mentioned "Max number of allowed recovery by s3 when fault happens and motr error's out with specific error code" in s3config file